escrow/contracts/Escrow.cdc

```diff
@@ -0,0 +1,192 @@
+/*
+    Escrow Contract for managing NFTs in a Leaderboard Context.
+    Holds NFTs in Escrow account awaiting transfer or burn.
+
+    Authors:
+        Corey Humeston: corey.humeston@dapperlabs.com
+        Deewai Abdullahi: innocent.abdullahi@dapperlabs.com
+*/
+
+import NonFungibleToken from "./NonFungibleToken.cdc"
+
+pub contract Escrow {
+    // Event emitted when a new leaderboard is created.
+    pub event LeaderboardCreated(name: String)
+
+    // Event emitted when an NFT is deposited to a leaderboard.
+    pub event NFTDeposited(leaderboardName: String, nftID: UInt64, owner: Address)
+
+    // Event emitted when an NFT is withdrawn from a leaderboard.
+    pub event NFTWithdrawn(leaderboardName: String, nftID: UInt64, owner: Address)
+
+    // Event emitted when an NFT is burned from a leaderboard.
+    pub event NFTBurned(leaderboardName: String, nftID: UInt64)
```

Comment:
i think we should rename this to EntryBurned event so it doesn't conflict with the nft's burned event in the AllDay contract

Reply:
Good idea.